java/runtime/src/main/java/org/ray/runtime/RayNativeRuntime.java

```diff
@@ -46,6 +48,10 @@
     for (String library : libraries) {
       String fileName = System.mapLibraryName(library);
       try (FileUtil.TempFile libFile = FileUtil.getTempFileFromResource(fileName)) {
+        // Expose ray ABI symbols which may be dependent by other shared
+        // libraries such as libstreaming_java.so.
```

Comment:
who depends on who? Do you mean: "Expose ray ABI symbols that other shared libraries, such as libstreaming_java.so, might be dependent on" ?

Reply:
That's right

Reply:
> That's right

Could you update comment?

Reply:
fixed